nodebuilder/config.go

```diff
@@ -67,7 +68,7 @@ func SaveConfig(path string, cfg *Config) error {
 	if err != nil {
 		return err
 	}
-	defer f.Close()
+	defer f.Close() //nolint: errcheck
```

Comment:
I know this code could be a problem, but I copied it form the old codes, I'm not sure if I should correct all of them because I do not fully understand the logic of old codes. Sure, I can fix at least my code if necessary, please tell me your opinion.

Reply:
Fixed, please review.